src/js/shop/components/Checkout/AddressFields.js

```diff
@@ -0,0 +1,154 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { FormattedMessage } from "react-intl";
+import Select from "react-select";
+import { country_list, SUPPORTED_COUNTRIES } from "./constants";
+
+import { FormField, FormGroup } from "./FormFields";
+
+const AddressFields = ({
+    prefix,
+    company = "",
+    chamberOfCommerce = "",
+    street = "",
+    number = "",
+    city = "",
+    postalCode = "",
+    country = { value: "", label: "" },
```

Comment:
Might be a good idea to make some extra checks in the code, because in this case the default value will be applied only if the `country` prop is `undefined`. If it's `null` or an empty object, the default value is not applied.